packages/components/src/button/style.scss

```diff
@@ -119,13 +119,8 @@
 			background-size: 100px 100%;
 			// Disable reason: This function call looks nicer when each argument is on its own line.
 			/* stylelint-disable */
-			background-image: linear-gradient(
-				-45deg,
-				theme(button) 28%,
-				color(theme(button) shade(20%)) 28%,
-				color(theme(button) shade(20%)) 72%,
-				theme(button) 72%
-			);
+			background-image:
```

Comment:
Same as before, the multi-line gradient syntax were added for a reason, which is, weirdly, counter to the previous reason :)